### PR TITLE
[React Components] Refresh the page after Sign In / Sign Out

### PR DESCRIPTION
**Pull request by alkim91**
## Description

For Signing Out :
I added the baseUrl(Storefront URL) in the App configs, and in the event handler for signing out, we redirect the user to the main storefront once they click 'Sign Out'. The baseUrl is configured based on the main storefront url(Storefront can be any language/country, not always US and English), and implemented in StoreConfigExporter.

For Signing In :
Added a simple refresh after all the sign-in flow is complete.

## Related Issue

https://github.com/adobe/aem-cif-guides-venia/pull/59

## Motivation and Context

If users are in any admin pages(Address book, account management), once they sign out we want them to be redirected back to the main storefront. Currently, we revoke the user token and reset cookies as well as the contexts, but no refresh/redirect takes place.

## How Has This Been Tested?

This has been tested in the two sign out user cases.

1. Signing out from the main navigation. Once we click 'sign out', no matter which page we're in(Catalog, address book, etc), we're taken back to the storefront.

2. Same thing happens when signing out from the account panel in responsive mode.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: react-components/src/components/MyAccount/myAccount.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ import { useConfigContext } from '../../context/ConfigContext';
 const MyAccount = props => {
     const { showMenu, showChangePassword, showAccountInformation } = props;
     const [{ currentUser, inProgress }, { signOut }] = useUserContext();
-    const { pagePaths } = useConfigContext();
```
Comment (herzog31) [inline react-components/src/components/MyAccount/myAccount.js:30]: Please remove the unused variable `storeView` here.

**Review by herzog31 — APPROVED** [review]

File: react-components/src/components/MyAccount/myAccount.js (hunk 1 above)
Comment (alkim91) [reply react-components/src/components/MyAccount/myAccount.js:30]: Removed!

File: react-components/src/components/MyAccount/myAccount.js [hunk 2]
```diff
@@ -39,12 +39,20 @@ const MyAccount = props => {
...
+    };
+
     const handleSignOut = async () => {
         dispatch({ type: 'reset' });
         await signOut();
         if (showMenu) {
             showMenu();
         }
+
+        if (pagePaths && pagePaths.baseUrl) {
```
Comment (herzog31) [inline react-components/src/components/MyAccount/myAccount.js:53]: I would move the check `pagePaths && pagePaths.baseUrl` into the `redirectBacktoStorefront` method at line 42, so you check the variable immediately before usage.

Comment (alkim91) [reply react-components/src/components/MyAccount/myAccount.js:53]: Good call. Moved.

**Review by herzog31 — APPROVED** [review]

File: examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/components/page/page.html [hunk 3]
```diff
@@ -13,36 +13,45 @@
...
-        <sly data-sly-test="${!isRedirectPage}">
-            <sly data-sly-include="body.socialmedia_begin.html"></sly>
-            <sly data-sly-include="body.html"></sly>
-            <sly data-sly-call="${footer.footer @ page = page}"></sly>
-            <sly data-sly-include="body.socialmedia_end.html"></sly>
-        </sly>
-    </body>
+<!DOCTYPE html>
+<html
+  data-sly-use.page="com.adobe.cq.wcm.core.components.models.Page"
```
Comment (dplaton) [inline examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/components/page/page.html:18]: Four spaces indentation!

**Review by dplaton — APPROVED** [review]
Please fix the indentation of the `page.html` file.

File: examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/components/page/page.html (hunk 3 above)
Comment (alkim91) [reply examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/components/page/page.html]: Sorry - Fixed :)

**Pull request closed without merging**